test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Fill out test script to encompass typescript testing, client build, docker set up etc

Reply:
Remove docker-compose down in test script and prefix test tables and add supporting function to remove test tables